runtime/gc_glue_java/ObjectModel.hpp

```diff
@@ -562,6 +562,20 @@ class GC_ObjectModel : public GC_ObjectModelBase
 	{
 		GC_ObjectModelBase::fixupForwardedObject(forwardedHeader, destinationObjectPtr, objectAge);
 
+#if defined(J9VM_ENV_DATA64)
+		if (_javaVM->isIndexableDataAddrPresent) {
+			if (isIndexable(forwardedHeader)) {
+				/**
```

Comment:
we could remove this comment. (perhaps it made sense in case of double mapping which we don't have any more)